setup.py

```diff
@@ -0,0 +1,65 @@
+from pkg_resources import parse_version
+from configparser import ConfigParser
+import setuptools
+
+assert parse_version(setuptools.__version__) >= parse_version("36.2")
+
+# note: all settings are in settings.ini; edit there, not here
+config = ConfigParser(delimiters=["="])
+config.read("settings.ini")
+cfg = config["DEFAULT"]
+
+cfg_keys = "version description keywords author author_email".split()
+expected = (
+    cfg_keys
+    + "lib_name user branch license status min_python audience language".split()
+)
+for o in expected:
+    assert o in cfg, "missing expected setting: {}".format(o)
+setup_cfg = {o: cfg[o] for o in cfg_keys}
+
+licenses = {
+    "mit": ("MIT License", "MIT License (MIT)",),
+}
+statuses = [
+    "1 - Planning",
+    "2 - Pre-Alpha",
+    "3 - Alpha",
+    "4 - Beta",
+    "5 - Production/Stable",
+    "6 - Mature",
+    "7 - Inactive",
+]
+py_versions = (
+    "2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7 3.0 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8".split()
```

Comment:
you can simply write python 2 and python 3 versions like >=python3.0 etc

Reply:
Removed this, it was redundant